<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>nuevaPersona</class>
 <widget class="QDialog" name="nuevaPersona">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>281</width>
    <height>224</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nueva persona</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::ExpandingFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="lblNombre">
       <property name="text">
        <string>Nombre:</string>
       </property>
       <property name="buddy">
        <cstring>txtNombre</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="txtNombre"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblCedula">
       <property name="text">
        <string>Cédula:</string>
       </property>
       <property name="buddy">
        <cstring>txtCedula</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="txtCedula"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lbltelefono">
       <property name="text">
        <string>Teléfono:</string>
       </property>
       <property name="buddy">
        <cstring>txtTelefono</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="txtTelefono"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lblDireccion">
       <property name="text">
        <string>Dirección:</string>
       </property>
       <property name="buddy">
        <cstring>txtDireccion</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="txtDireccion"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="lblCorreo">
       <property name="text">
        <string>Correo:</string>
       </property>
       <property name="buddy">
        <cstring>txtCorreo</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="txtCorreo"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="lblNotas">
       <property name="text">
        <string>Notas:</string>
       </property>
       <property name="buddy">
        <cstring>txtNotas</cstring>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="txtNotas"/>
     </item>
    </layout>
   </item>
   <item row="3" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>txtNombre</tabstop>
  <tabstop>txtCedula</tabstop>
  <tabstop>txtTelefono</tabstop>
  <tabstop>txtDireccion</tabstop>
  <tabstop>txtCorreo</tabstop>
  <tabstop>txtNotas</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>nuevaPersona</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>nuevaPersona</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
